src/server/cluster/outgoing_slot_migration.cc

```diff
@@ -240,6 +248,9 @@ void OutgoingMigration::SyncFb() {
 }
 
 bool OutgoingMigration::FinalizeMigration(long attempt) {
+  OnAllShards([this](auto& migration) { server_family_->CancelBlockingOnThread(); });
```

Comment:
why do we cancel all blocking commands if we migrate only specific slots? we should cancel only the blocking commands only on this slots